types/relay-runtime/relay-runtime-tests.tsx

```diff
@@ -16,6 +16,9 @@ import {
 
 const source = new RecordSource();
 const store = new Store(source);
+const storeWithOptions = new Store(source, {
+    gcReleaseBufferSize: 10,
+});
```

Comment:
I suppose the combinations here could be more thorough.  Not sure how far we generally want to go with test cases.  If we're going for completeness, we should do a bit more here.

Reply:
I added some more tests. There are many possible combinations though so I'd keep it simple unless there is some problem I didn't think about. :)